src/frontend/screens/Settings/ProjectSettings/YourTeam/ReviewInvitation.tsx

```diff
@@ -0,0 +1,104 @@
+import {StyleSheet, View} from 'react-native';
+import {Text} from '../../../../sharedComponents/Text';
+import {defineMessages, useIntl} from 'react-intl';
+import {NativeNavigationComponent} from '../../../../sharedTypes';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import React from 'react';
+import {WHITE} from '../../../../lib/styles';
+import {Button} from '../../../../sharedComponents/Button';
+import {DeviceNameWithIcon} from '../../../../sharedComponents/DeviceNameWithIcon';
+import {RoleWithIcon} from '../../../../sharedComponents/RoleWithIcon';
+import {useProject} from '../../../../hooks/server/projects';
+import {WaitingForInviteAccept} from './WaitingForInviteAccept';
+import {useBottomSheetModal} from '../../../../sharedComponents/BottomSheetModal';
+import {MEMBER_ROLE_ID} from './SelectInviteeRole';
+
+const m = defineMessages({
+  title: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.ReviewInvitation.title',
+    defaultMessage: 'Review Invitation',
+  },
+  youAreInviting: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.ReviewInvitation.youAreInviting',
+    defaultMessage: 'You are inviting:',
+  },
+  coordinator: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.ReviewInvitation.coordinator',
+    defaultMessage: 'Coordinator',
+  },
+  participant: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.ReviewInvitation.participant',
+    defaultMessage: 'Participant',
+  },
+  sendInvite: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.ReviewInvitation.sendInvite',
+    defaultMessage: 'Send Invite',
+  },
+});
+
+export const ReviewInvitation: NativeNavigationComponent<
+  'ReviewInvitation'
+> = ({route, navigation}) => {
+  const {formatMessage: t} = useIntl();
+  const {role, ...rest} = route.params;
+  const [inviteSent, setInviteSent] = React.useState(false);
+  const {openSheet, ...restBottomSheet} = useBottomSheetModal({
+    openOnMount: false,
+  });
+  const project = useProject();
+
+  function sendInvite() {
+    setInviteSent(true);
+    project.$member
+      .invite(rest.deviceId, {roleId: role})
+      .then(val => navigation.navigate('InviteAccepted', {...route.params}))
```

Comment:
This is timing out after 5 seconds:

`Error: Server timed out after 5000ms.
            The server could be closed or the transport is down.`
            
Otherwise, the other device is recieving the invite and is able to accept it and join the project @gmaclennan